src/test/java/seedu/address/logic/parser/RemoveCommandParserTest.java

```diff
@@ -0,0 +1,4 @@
+package seedu.address.logic.parser;
+
+public class RemoveCommandParserTest {
```

Comment:
I assume this is placeholder, to be added later?

Reply:
Yes, I will be added in soon

Reply:
Perhaps include a test regarding resetting all tags? Aside from that, seems alright to me 👍